src/index.html

```diff
@@ -4,9 +4,315 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__ltem">
+            <a href="#Apple" class="is-active">Apple</a>
+          </li>
+          <li class="nav__ltem">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__ltem">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__ltem">
+            <a href="#Laptops & Computers" data-qa="nav-hover"
+             class="nav__link">
+              Laptops & Computers
+            </a>
```

Comment:
If tag has more than 2 attributes each attribute should be on new line/ Closed bracket should be on same line as open. Fix all similar cases
```suggestion
            <a
              href="#Laptops & Computers"
              data-qa="nav-hover"
              class="nav__link"
            >
              Laptops & Computers
            </a>
```